server/schedule/checker/replica_checker.go

```diff
@@ -62,35 +62,46 @@ func (r *ReplicaChecker) GetType() string {
 }
 
 // Check verifies a region's replicas, creating an operator.Operator if need.
-func (r *ReplicaChecker) Check(region *core.RegionInfo) *operator.Operator {
+func (r *ReplicaChecker) Check(region *core.RegionInfo) (op *operator.Operator) {
 	checkerCounter.WithLabelValues("replica_checker", "check").Inc()
-	if op := r.checkDownPeer(region); op != nil {
+	if op = r.checkDownPeer(region); op != nil {
 		checkerCounter.WithLabelValues("replica_checker", "new-operator").Inc()
-		op.SetPriorityLevel(core.HighPriority)
-		return op
 	}
-	if op := r.checkOfflinePeer(region); op != nil {
-		checkerCounter.WithLabelValues("replica_checker", "new-operator").Inc()
-		op.SetPriorityLevel(core.HighPriority)
-		return op
+	if miss, storeID := r.checkOfflinePeer(region); miss > 0 {
+		if op == nil {
+			checkerCounter.WithLabelValues("replica_checker", "new-operator").Inc()
```

Comment:
Why the metrics increase when `op == nil`?

Reply:
yes, it will place to if op!=nil